Sources/Components/Broadcast/BroadcastContainerView/BroadcastContainerView.swift

```diff
@@ -0,0 +1,258 @@
+//
+//  Copyright © FINN.no AS, Inc. All rights reserved.
+//
+
+import UIKit
+
+public protocol BroadcastContainerViewDelegate: class {
+    /// Called when the BroadcastContainerView is about to display its broadcasts
+    ///
+    /// - Parameters:
+    ///   - broadcastContainerView: the BroadcastContainerView
+    ///   - containerSize: the size that the BroadcastContainerView will have when the broadcasts are displayed
+    ///   - commitToDisplaying: a closure that must be called in order for the BroadcastContainerView to display its broadcasts
+    func broadcastContainer(_ broadcastContainerView: BroadcastContainerView, willDisplayBroadcastsWithContainerSize containerSize: CGSize, commitToDisplaying: @escaping (() -> Void))
+
+    /// Called when the BroadcastContainerView is about to dismiss a broadcast
+    ///
+    /// - Parameters:
+    ///   - broadcastContainerView: the BroadcastContainerView
+    ///   - index: the index of the broadcast that is about to be dismissed
+    ///   - newContainerSize: the size that the BroadcastContainerView will have when the broadcast is dismissed
+    ///   - commitToDismissal: a closure that must be called in order for the BroadcastContainerView to dismiss the broadcast
+    func broadcastContainer(_ broadcastContainerView: BroadcastContainerView, willDismissBroadcastAtIndex index: Int, withNewContainerSize newContainerSize: CGSize, commitToDismissal: @escaping (() -> Void))
+}
+
+public protocol BroadcastContainerViewDataSource: class {
+    /// The number of broadcasts to display in the BroadcastContainerView
+    ///
+    /// - Parameter broadcastContainerView: the BroadcastContainerView
+    /// - Returns: The number of broadcasts to display in the BroadcastContainerView
+    func numberOfBroadcastsToDisplay(in broadcastContainerView: BroadcastContainerView) -> Int
+
+    /// The broadcast to display at the index
+    ///
+    /// - Parameters:
+    ///   - broadcastContainerView: the BroadcastContainerView
+    ///   - index: the index for the broadcast
+    /// - Returns: The broadcast to display at the index
+    func broadcastContainerView(_ broadcastContainerView: BroadcastContainerView, broadcastForIndex index: Int) -> Broadcast
+}
+
+/// A view that is used to display multiple BroadcastViews
+public final class BroadcastContainerView: UIView {
+    private struct LayoutConstants {
+        struct ContentView {
+            static let insets = Insets(top: .mediumLargeSpacing, leading: .mediumLargeSpacing, bottom: 0, trailing: .mediumLargeSpacing)
```

Comment:
I usually use structs as models that hold data that is meant to be transporter or encapsulated, not sure if LayoutConstants.ContentView.insets would be a good candidate for this instead of for example having 

```
private let contentViewInsets = Insets(...
```

Same for the BroadcastView.spacing

Reply:
Sure 👍  I just like it this way because of the readability that the name spacing of the nested structs and properties gives 😊 To me, when I read LayoutConstants.ContentView.insets, I know that it is LayoutConsants and that it is for the ContentView and that it's the insets for the ContentView 😊 Of course the LayoutConsants only gives meaning to have because there's layout constants for the Broadcast view spacing as well.

Reply:
Change pushed 😊

Reply:
🔥 sweet!